src/js/jsx/sections/transform/TransformPanel.jsx

```diff
@@ -53,9 +53,8 @@ define(function (require, exports, module) {
 
         shouldComponentUpdate: function (nextProps, nextState) {
             return this.state.referencePoint !== nextState.referencePoint ||
-                this.props.active !== nextProps.active ||
                 this.props.disabled !== nextProps.disabled ||
-                !Immutable.is(this.props.document, nextProps.document);
+                !Immutable.is(this.props.document.layers.selected, nextProps.document.layers.selected);
```

Comment:
It's possible that this will cause us to miss a necessary render when the document bounds change (which is shown in this panel when no layers are selected). So, maybe also compare the document bounds?